v4/source/introduction.yaml

```diff
@@ -243,7 +412,8 @@ tags:
       - preferences_deleted
       - reaction_added
       - reaction_removed
-      - response
+      - r
+        esponse
```

Comment:
Accidental new line?

Reply:
Also is `response` a valid event type?

Reply:
> Accidental new line?
Yea.

> Also is response a valid event type?
I didn't add `response`. It was already there.